Symfony/CS/Tokenizer/Token.php

```diff
@@ -441,14 +443,10 @@ public function toArray()
 
     public function toJSON()
```

Comment:
Renaming it isn't a bc break i don't think given that php isn't case sensitive with function calls.